[18.0][FIX] sale_stock_return_request: Skip test

victoralmau opened the pull request:
Skip tests to unlock CI

Related to https://github.com/OCA/stock-logistics-request/pull/85#issuecomment-3597401441 + https://github.com/OCA/sale-workflow/pull/4051

Please @pedrobaeza can you review it?

@Tecnativa

victoralmau commented:
Ping @JoanSForgeFlow

pedrobaeza commented:
But my change was correct... It has been the other change which breaks the CI, isn't it?

victoralmau commented:
> But my change was correct... It has been the other change which breaks the CI, isn't it?

https://github.com/OCA/stock-logistics-request/pull/85#issuecomment-3597401441

pedrobaeza commented:
Yes, but then the tests should be adapted, not to revert a previous commit. I also simplify all the stuff, which was cluttered. Please just adapt instead of reverting.

victoralmau committed:
[FIX] sale_stock_return_request: Skip test

Related to https://github.com/OCA/stock-logistics-request/pull/85#issuecomment-3597401441

victoralmau commented:
> Yes, but then the tests should be adapted, not to revert a previous commit. I also simplify all the stuff, which was cluttered. Please just adapt instead of reverting.

I don't know the module and do not have time to analyze and find the right solution, so in order to block the CI, I have commented out the test.

rousseldenis reviewed (commented):
@victoralmau I'd prefer the usage of `@unittest.skip` in order to let a warning in logs

pedrobaeza commented:
But the warnings are now provoking red CIs, isn't it?

rousseldenis commented:
> But the warnings are now provoking red CIs, isn't it?

You can add a line here too: https://github.com/OCA/sale-workflow/blob/18.0/checklog-odoo.cfg#L3

victoralmau commented:
It's ready and CI is green.

pedrobaeza reviewed (approved):
/ocabot merge patch
